static/src/stylesheets/module/experiments/_survey.scss

```diff
@@ -0,0 +1,74 @@
+.impressions-survey__heading {
+  padding: .5em 1em;
```

Comment:
Not sure if this design is following our grid system, but these values might help: https://github.com/guardian/guss-grid-system